pkg/process/checks/container.go

```diff
@@ -58,6 +59,11 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 		return nil, err
 	}
 
+	networkID, err := util.GetNetworkID()
```

Comment:
Could we possibly get this in `Init()` or is there caching and race conditions at play?

Reply:
We could get it in `Init()`.